integration-test/1491-peruvian-shields.py

```diff
@@ -0,0 +1,135 @@
+# -*- encoding: utf-8 -*-
+from . import FixtureTest
+
+
+class PeruShieldTest(FixtureTest):
+    def test_peayroads(self):
+        import dsl
+
+        z, x, y = (16, 19261, 35238)
+
+        self.generate_fixtures(
+            dsl.is_in('PE', z, x, y),
+            # https://www.openstreetmap.org/way/414535595
+            dsl.way(414535595, dsl.tile_diagonal(z, x, y), {
+                'highway': u'primary',
+                'ref': u'AY-104',
+                'source': u'openstreetmap.org',
+            }),
+            dsl.relation(1, {
+                'network': u'PE-AY-roads',
+                'ref': u'AY-104',
+                'route': u'road',
+                'source': u'openstreetmap.org',
+                'type': u'route',
+            }, ways=[414535595]),
+        )
+
+        self.assert_has_feature(
+            z, x, y, 'roads', {
+                'id': 414535595,
+                'shield_text': '104',
+                'network': u'PE:AY',
```

Comment:
Please comment here all Peru admin-1 region codes so those can be moved to client-side lookup versus regEx.

Reply:
All state codes listed here: https://github.com/tilezen/vector-datasource/blob/ec7b47c91b69728631778ffe258651ebdb723df2/vectordatasource/transform.py#L5371-L5396